packages/active-campaign-client/src/helpers/manageListSubscription.ts

```diff
@@ -0,0 +1,40 @@
+import { APIGatewayProxyResult } from 'aws-lambda';
+import { acClient } from '../clients/activeCampaignClient';
+
+export async function manageListSubscription(
+  cognitoUserId: string,
+  listName: string,
+  action: 'subscribe' | 'unsubscribe'
```

Comment:
I think is better to split this file in two seperate helpers one for subscribe and another for unsubscribe somthing like:
- `addContactToList`
- `removeContactToList`